ui/src/app/ui-banner/ui-banner.tsx

```diff
@@ -66,6 +68,12 @@ export const Banner = (props: React.Props<any>) => {
                         chatBottomPosition = 85;
                     }
                 }
+                try {
+                    const externalLink = new ExternalLink(chatUrl);
+                    chatUrl = externalLink.ref;
+                } catch (InvalidExternalLinkError) {
+                    chatUrl = 'invalid-url';
```

Comment:
Could we use `null` instead of a magic string? Effectively the same, just seems a bit more expressive.  :-)

Reply:
The null check is added to not render a button when there is no chat url provided. If it is invalid then should we give them a warning/message or remove the button entirely?

Reply:
Ahh. Yeah, I think leaving the button with a warning message is fine!